### PR TITLE
 Increase CCM workers to speed up node bootstrap process

### PR DESCRIPTION
**Pull request by hakuna-matatah**

### CODE REVIEW

**Review by rifelpet — APPROVED** [review]

**Comment by hakman** [comment]
Seeing rate limiting in [CCM logs](https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kops/16256/presubmit-kops-aws-scale-amazonvpc-using-cl2/1747737480699842560/artifacts/cluster-info/kube-system/aws-cloud-controller-manager-xstk7/aws-cloud-controller-manager.log) with `30`. Let's set only to `10` for now.

File: tests/e2e/scenarios/scalability/run-test.sh [hunk 1]
```diff
@@ -95,7 +95,7 @@ create_args+=("--image=${INSTANCE_IMAGE:-ssm:/aws/service/canonical/ubuntu/serve
 create_args+=("--set spec.etcdClusters[0].manager.listenMetricsURLs=http://localhost:2382")
 create_args+=("--set spec.etcdClusters[0].manager.env=ETCD_QUOTA_BACKEND_BYTES=8589934592")
 create_args+=("--set spec.etcdClusters[1].manager.env=ETCD_QUOTA_BACKEND_BYTES=8589934592")
-create_args+=("--set spec.cloudControllerManager.concurrentNodeSyncs=5")
+create_args+=("--set spec.cloudControllerManager.concurrentNodeSyncs=30")
```
Comment (hakman) [inline tests/e2e/scenarios/scalability/run-test.sh:98]: I think this should be enough for now. There's no additional speed going further.
```suggestion
create_args+=("--set spec.cloudControllerManager.concurrentNodeSyncs=10")
```

**Review by hakman — CHANGES_REQUESTED** [review]

File: tests/e2e/scenarios/scalability/run-test.sh (hunk 1 above)
Comment (hakuna-matatah) [reply tests/e2e/scenarios/scalability/run-test.sh:98]: @hakman did you notice client-side rate liming when you were testing out with `20` ?

**Review by hakman — APPROVED** [review]

File: tests/e2e/scenarios/scalability/run-test.sh (hunk 1 above)
Comment (hakman) [reply tests/e2e/scenarios/scalability/run-test.sh]: no, but i also did not notice any visible gains

**Comment by hakuna-matatah** [comment]
@hakman I think we can merge these changes.

**Pull request merged**